### PR TITLE
Clean Build for 2019

### LINKED ISSUE #2343: Build Issues with 37833fcf2ea4d1f54d0e846c78ff79b307509809
It's been a while since I've worked on this project and I wanted to get back into it.  I started with a fresh development system with only **git version 2.20.1.windows.1** and **Visual Studio Community 2017 15.9.7** installed.  I wanted to approach it as if I was a newbie.  

I've encountered a series of problems with a clean build.  I'm going to use this issue as a way of tracking the individual problems found.

### PR DESCRIPTION
**Pull request by c0g1t8**
I attempted to rebuild the web only portion of the allReady on a fresh development system and encountered multiple build issues.  This PR should resolve #2343(#2344, #2345, #2346)

### CODE REVIEW

**Comment by c0g1t8** [comment]
Hmmm.... interesting.  I'm seeing errors running the _AllReady.ScenarioTest_ locally.  I'm going to spend some time investigating.

**Comment by c0g1t8** [comment]
Appveyor appears to have had some sporadic issues restoring npm packages.  This caused some non-reproducible failures.  They seemed to have been resolved by the end of today.

Examination of the output from Appveyor indicated that is was using _Gulp 4.0_.  So I updated gulpfile.js to be compatible.  This worked locally.

As for the integration test, the local chromedriver version was 2.33 while appveyor was using 2.45.  So local results didn't match up.  I think I have a stable fix for that after a couple of hours of trial and error.

File: AllReadyApp/Web-App/AllReady/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "gulp-cssmin": "0.1.7",
     "gulp-sourcemaps": "1.6.0",
     "gulp-typescript": "3.1.5",
+    "natives": "1.1.6",
```
Comment (jonparker) [inline AllReadyApp/Web-App/AllReady/package.json]: I read some of the notes and warnings about this package behaving differently on different versions of NodeJS.

Comment (c0g1t8) [reply AllReadyApp/Web-App/AllReady/package.json]: I ran across the same but I think the best long term solution is simply move to Gulp 4.0.  I’ve got a separate PR for that in the pipeline. Decided it was better to piecemeal out things.

There are a lot of packages.  Not sure if they are all being used either.  NPM audit shows a bunch of stuff well above my pay grade :)

**Comment by c0g1t8** [comment]
I'm closing this PR.

Why?
- _Node 8.15_ is being used on the build server.  Degrading from _Node 10_ resolves the _Gulp_ related issues.  See #2351 
- _AllReady.ScenarioTest_ is so old it's difficult to _'fix'_.  It was much easier to simply upgrade the whole project to _Core 2.1_ and _Canopy 2_.  Also suspect that some of the underlying instability and inconsistency could have been due to issues in _Canopy_ and _Selenium_.  And upgrading everything should make it much easier to support.

**Pull request closed without merging**